AutoInstaller.ps1

```diff
@@ -1,33 +1,36 @@
-
-
 # Check user has webhook URL ready
 $userPrompt = Read-Host -Prompt "Do you have your webhook URL ready? Y/N"
 if ($userprompt -ne 'Y') {
     Write-Output "Please create a Discord webhook before continuing."
     exit
 }
-    #Create Directory structure
-    New-Item C:\VeeamScripts -Type directory
-    $downloadlocation = Read-Host -Prompt "Please Type the full path of your download location including Drive letter"
-    Expand-Archive $downloadlocation -DestinationPath C:\VeeamScripts
-    Rename-Item C:\VeeamScripts\VeeamDiscordNotifications-v1.5 C:\VeeamScripts\VeeamDiscordNotifications
+#Create Directory structure
+New-Item C:\VeeamScripts -Type directory# Get latest release from GitHub.
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$latestRelease = Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/latest -Headers @{"Accept"="application/json"} -UseBasicParsing
+# Release IDs are returned in a format of {"id":3622206,"tag_name":"v1.0"} so we need to extract tag_name.
+$latestVersion = $latestRelease.Content | ConvertFrom-Json | Select-Object tag_name -ExpandProperty tag_name
+# Pull latest version of script from GitHub
+Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/download/$LatestVersion/VeeamDiscordNotifications-$LatestVersion.zip -OutFile $PSScriptRoot\VeeamDiscordNotifications-$LatestVersion.zip
+# Expand downloaded ZIP
+Expand-Archive $PSScriptRoot\VeeamDiscordNotifications-$LatestVersion.zip -DestinationPath C:\VeeamScripts
 
 #Assign webhook url to variable
 $webhookurl = Read-Host -Prompt "Please paste your Webhook URL now"
 
-#Read Config File & Write the user webhook
+#Get the config file and write the user webhook
 $Config = (Get-Content C:\VeeamScripts\VeeamDiscordNotifications\config\conf.json) | ConvertFrom-Json
 $Config.webhook = $webhookurl
 #Write Config
 ConvertTo-Json $Config | Set-Content C:\VeeamScripts\VeeamDiscordNotifications\config\conf.json
- # Unblock script files
- Unblock-File C:\VeeamScripts\VeeamDiscordNotifications\DiscordNotificationBootstrap.ps1
- Unblock-File C:\VeeamScripts\VeeamDiscordNotifications\DiscordVeeamAlertSender.ps1
+#Unblock script files
+Unblock-File C:\VeeamScripts\VeeamDiscordNotifications\DiscordNotificationBootstrap.ps1
+Unblock-File C:\VeeamScripts\VeeamDiscordNotifications\DiscordVeeamAlertSender.ps1
 Unblock-File C:\VeeamScripts\VeeamDiscordNotifications\resources\logger.psm1
 Unblock-File C:\VeeamScripts\VeeamDiscordNotifications\UpdateVeeamDiscordNotification.ps1
 
 

```

Comment:
Nothing necessarily wrong with a double line break for readability but it's not very consistent with the rest of the script.